docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TreeFeaturization/FastForestBinaryFeaturizationWithOptions.cs

```diff
@@ -58,7 +58,8 @@ public static void Example()
             };
 
             // Define the tree-based featurizer's configuration.
-            var options = new FastForestBinaryFeaturizationEstimator.Options
+            var options = new FastForestBinaryFeaturizationEstimator
+    .Options
```

Comment:
>.Options [](start = 4, length = 8)

was this crossing 85 character limit?

Reply:
Sorry that's a mistake, I'll revert that. Thanks for pointing it out.

---
In reply to: [304142959](https://github.com/dotnet/machinelearning/pull/4009#discussion_r304142959) [](ancestors = 304142959)